grails-app/services/streama/TheMovieDbService.groovy

```diff
@@ -20,8 +20,16 @@ class TheMovieDbService {
 
 
   def getSimilarMovies(movieId){
-    def JsonContent = new URL(BASE_URL + "/movie/$movieId/similar?api_key=$API_KEY").text
-    return new JsonSlurper().parseText(JsonContent)
+    def JsonContent_similar = new URL(BASE_URL + "/movie/$movieId/similar?api_key=$API_KEY").text
+    def json_similar = new JsonSlurper().parseText(JsonContent_similar)
+    for(item in json_similar.results ){
```

Comment:
I'd prefer if the variable be named in camelCase instead of snake_case. Snake_case is only used for properties that are taken 1to1 from the API